python/lsst/obs/base/_instrument.py

```diff
@@ -111,6 +118,29 @@ def getName(cls):
         """
         raise NotImplementedError()
 
+    @classmethod
+    def getCuratedCalibrationNames(cls) -> Set[str]:
```

Comment:
Should this be a method, or a property?

Reply:
Class properties are non-standard since you can't simply add `@property` to `@classmethod`. Astropy have written a decorator that would do that properly but we've never used it before.